src/ground-truth.rkt

```diff
@@ -14,16 +15,26 @@
 
 (define ground-truth-require-convergence (make-parameter #t))
 
+(define (prog->spec prog ctx)
+  (resugar-program prog ctx #:full #f))
+
 ;; Returns a function that maps an ival to a list of ivals
 ;; The first element of that function's output tells you if the input is good
 ;; The other elements of that function's output tell you the output values
 (define (make-search-func pre exprs ctxs)
   ; (eprintf "pre ~a, expers ~a, ctxs ~a\n" pre exprs ctxs)
-  (define fns (compile-progs (cons pre exprs) 'ival (car ctxs)))
+  (define specs
+    (for/list ([spec (cons pre exprs)])
+      (prog->spec spec (car ctxs))))
+  (define fns (compile-specs specs (context-vars (car ctxs))))
```

Comment:
So I sort-of forget what this code is even doing but why `(car ctxs)`? Whatever, I guess